### ISSUE TITLE
Support for genotype data in VCF format

### ISSUE THREAD

**Issue by nevrome** [poseidon-framework/poseidon-schema#45]
The VCF file format appears to be a popular, powerful and (comparatively) well specified file format for genotype data. Poseidon could (one day!) support it the same way it supports Packed PLINK and EIGENSTRAT data. Some observations:

- VCF files seem to be very flexible and capable of storing a lot more information than PLINK or EIGENSTRAT files. That makes them harder to parse and render. Most importantly there is no lossless conversion between the formats, given VCF's greater flexibility.
- The VCF file definition seems to be adjusted relatively frequently. [v.4.3](https://github.com/samtools/hts-specs/blob/master/VCFv4.3.pdf) is published, [v.4.4](https://github.com/samtools/hts-specs/blob/master/VCFv4.4.draft.pdf) on the way. For Poseidon we would have to decide which version we support and keep track of the changes in the format.
- For poseidon-hs: `sequence-formats` already [supports it](https://hackage.haskell.org/package/sequence-formats-1.6.1/docs/SequenceFormats-VCF.html) (at least partially?). In case of missing functionality here, also [this script](https://github.com/Matthew-Mosior/Basic-Variant-Parser) or [this package](https://github.com/juanpaucar/vcf) may serve as an inspiration.

**Comment by stschiff** [comment]
Yes, VCF would have the sweet advantage of also encoding things like genotype likelihoods and read counts. Indeed! Wouldn't be a big problem to feature that, although `forge` would then have to make some choices, since VCF is a bit more general than Eigenstrat and Plink. Would have to be prepared to get feature requests then... but why not! I'll note it down as a nice addition. I already have plans to output read counts with pileupCaller, so might just actually output VCF.

**Comment by nevrome** [comment]
I'm closing issues that are already implemented in #93.
